swap/htlc.go

```diff
@@ -258,14 +378,20 @@ func (h *Htlc) AddTimeoutToEstimator(estimator *input.TxWeightEstimator) {
 	// Calculate maximum timeout witness size
 	//
 	// - number_of_witness_elements: 1 byte
-	// - sender_sig_length: 1 byte
+	// - sender_key_length: 1 byte
+	// - sender_key: 33 byte (htlc v2 only)
+	// - sender_key_length: 1 byte (htlc v2 only)
 	// - sender_sig: 73 bytes
-	// - zero_length: 1 byte
+	// - zero_length: 1 byte (htlc v1 only)
 	// - zero: 1 byte
 	// - witness_script_length: 1 byte
 	// - witness_script: len(script) bytes
 	maxTimeoutWitnessSize := 1 + 1 + 73 + 1 + 1 + 1 + len(h.Script)
```

Comment:
For clarity you could split this in two blocks and do the addition of the constant for both version separately. Also easier to eventually remove v1.

Reply:
done